scripts/static/2-produce-geo-files-static.R

```diff
@@ -4,48 +4,43 @@
 
 library(tidyverse)
 library(sf)
-library(jsonlite)
-library(testit)
-
-options(scipen = 999)
-
 
 #----Get counties around South Dakota --------------------------
 # We do this bc WAC is missing in 2017 for south dakota, 
 # which means RAC will be undercounted in surrounding area
 
 my_counties <- st_read("data/raw-data/big/counties.geojson")
 
+#Update 05/2021 - South Dakota was backfilled so this step is no longer needed
 
 # keep just south dakota
-south_dakota <- filter(my_counties %>% select(GEOID, STATEFP), STATEFP == "46")
+#south_dakota <- filter(my_counties %>% select(GEOID, STATEFP), STATEFP == "46")
```

Comment:
My vote here is to actually delete the code related to SD rather than comment it out. That will be saved in the Git history if people really want it and allows us to have "clean" code in the pubilc facing repo without chunks commented out. Any chance you could clean up this section a bit, maybe rename the entire section to "Get counties in AL from previous years"